libnmstate/schema.py

```diff
@@ -158,7 +162,46 @@ class BondMode(object):
     ALB = 'balance-alb'
 
 
-class LinuxBridge(object):
+DEPRECATED_CONSTANTS = {
+    'LinuxBridge.PORT_NAME': 'LinuxBridge.Port.NAME',
+    'LinuxBridge.PORT_STP_PRIORITY': 'LinuxBridge.Port.STP_PRIORITY',
+    'LinuxBridge.PORT_STP_HAIRPIN_MODE': 'LinuxBridge.Port.STP_HAIRPIN_MODE',
+    'LinuxBridge.PORT_STP_PATH_COST': 'LinuxBridge.Port.STP_PATH_COST',
+}
+
+
+class _NmstateContants(type):
```

Comment:
The class/type itself does not contain any information about deprecation, it is all in the separate dictionary. It can be used for all constants, even if they do not have anything that is deprecated. If there is another common code for all constant classes, it could go in there as well.
However, I do not care too much about the name, just wanted to share my thoughts about this.